index.js

```diff
@@ -1 +1,682 @@
-console.log("hello world")
\ No newline at end of file
+ let toggle=document.querySelector('.toggle')
+let header=document.querySelector('header')
+let dropdown=document.querySelector('.drop-down ')
+let socials=document.querySelector('.socials-ul')
+let secondRow=document.querySelector('.second-roww ');
+let secondRoww=document.querySelector('.second-row ');
+ toggle.addEventListener('click',translate)
+ function translate(){
+     dropdown.classList.toggle('nav-active');
+     secondRow.classList.toggle('nav-active');
+     socials.classList.toggle('nav-active');
+     header.classList.toggle('header');
+     if(secondRow.classList.contains('nav-active')){
+        //  console.log('hey')
+     }
+     else{
+        //  console.log('hi')
+        secondRow.classList.add('second-roww')
+        secondRow.classList.remove('second-row')
+     }
+     if(secondRow.classList.contains('nav-active')){
+         
+        secondRow.classList.remove('second-roww')
+       secondRow.classList.add('second-row')
+    }
+    else{
+        // console.log('hi')
+       secondRow.classList.add('second-roww')
+       secondRow.classList.remove('second-row')
+    }
+ }
+ 
+
+//  courosel
+var slideIndex = 1;
```

Comment:
I suggest using const or let, preferrably const unless you're altering your variable.

Reply:
okay i'd use const as my default from now